tools/lexical-model.ts

```diff
@@ -5,6 +5,7 @@ interface LexicalModel {
     allowedCharacters?: { initials?: string, medials?: string, finals?: string } | string,
     defaultBreakCharacter?: string
     sources?: Array<string>;
+    root?: string
```

Comment:
Throughout this PR, I'm unclear exactly what `root` is, and why we're accessing `root` everywhere. Could you please document this parameter?